build-targets-gradle-plugin/src/main/kotlin/com/faire/gradle/release/ComputeSourceFoldersTask.kt

```diff
@@ -27,43 +29,54 @@ import javax.inject.Inject
  */
 @CacheableTask
 internal abstract class ComputeSourceFoldersTask @Inject constructor(
-    objects: ObjectFactory,
+  objects: ObjectFactory,
 ) : DefaultTask() {
 
   @Suppress("UNUSED") // This is an input to the task, acting as a proxy for figuring out source files.
   @InputFiles
   @PathSensitive(PathSensitivity.RELATIVE)
   val buildFiles: ConfigurableFileCollection = objects.fileCollection()
-      .from(
-          project.rootProject.layout.projectDirectory.asFileTree.filter {
-            it.name == "build.gradle.kts" &&
-                "/test/" !in it.path // exclude test projects
-          },
-      )
+    .from(
+      project.rootProject.layout.projectDirectory.asFileTree.filter {
+        it.name == "build.gradle.kts" &&
+            "/test/" !in it.path // exclude test projects
```

Comment:
Should we include other test suites here too?

Reply:
We can add them via the exclude patterns, I don't think that generalizes to other users.

Reply:
Perhaps not, but ideally we could the JVM test suite plugin to find the sources of all test suites?

Reply:
Yep! We can definitely do that, I think we can do better in a follow-up.